dolphinscheduler-python/pydolphinscheduler/examples/yaml_define/DataX.yaml

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Define the workflow
+workflow:
+  name: "DataX"
+
+# Define the tasks under the workflow
+tasks:
+  - task_type: DataX
+    name: task
```

Comment:
General LGTM, but can we move `name` in the top of declare? I think it make more sense because `name` is unique in single task, like

```yaml
 - name: task
   task_type: DataX
   datasource_name: db
   datatarget_name: db
   sql: show tables;
   target_table: table_test
```